cmd/lakectl/cmd/branch_tools_test.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/treeverse/lakefs/pkg/metastore"
+	"github.com/treeverse/lakefs/pkg/metastore/mock"
+)
+
+const (
+	dbName     = "source"
+	repoName   = "repo"
+	branchName = "branch"
+)
+
+var (
+	validPath   = fmt.Sprintf("s3://%s/%s/path/to/schema", repoName, branchName)
+	invalidPath = fmt.Sprintf("s3/%s/%s/", repoName, branchName)
+)
+
+func TestExtractRepoAndBranchFromDBName(t *testing.T) {
+	type args struct {
+		ctx    context.Context
+		dbName string
+	}
+	tests := []struct {
+		name      string
+		args      args
+		want      string
+		want1     string
+		wantErr   bool
+		validPath bool
```

Comment:
Please export all fields in a `tests` struct.  It is often tempting to use`deep.Equal` or something else that relies on reflection on these at some later stage.  And that will compare no fields because Go reflection cannot see lowercase (unexported) fields.